### DISCUSSION TITLE
Générer entités à partir d'une bdd existante sur symfony 6

### DISCUSSION THREAD

**Discussion by tuwwix** [symfony/symfony#52013] [Q&A]
Bonjour, j'aimerai savoir comment générer des entités à partir d'une bdd existante sur symfony 6.
les commandes:
php bin/console doctrine:mapping:import --force AppBundle
symfony console doctrine:mapping:import "App\Entity" annotation --path=src/Entity
ne fonctionne pas surement par ce que doctrine:mapping:import ne tourne plus dans les nouvelles versions, est ce quelqu'un connait une alternative ? Merci.

**Comment by ghost** [comment]
English please
